doc/source/whatsnew/v0.21.0.txt

```diff
@@ -22,6 +22,8 @@ Check the :ref:`API Changes <whatsnew_0210.api_breaking>` and :ref:`deprecations
 New features
 ~~~~~~~~~~~~
 
+- New user-facing :class:`pandas.api.types.CategoricalDtype` for specifying
```

Comment:
move to the highlites (an provide the link to the sub-section); put the issue numbers there